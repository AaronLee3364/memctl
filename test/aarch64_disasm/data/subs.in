fffffff0070c0000 f10400a5 eb08012e 6b2a0108
