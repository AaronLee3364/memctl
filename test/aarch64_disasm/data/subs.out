fffffff0070c0000  SUBS    X5, X5, #0x100
fffffff0070c0004  SUBS    X14, X9, X8
fffffff0070c0008  SUBS    W8, W8, W10, UXTB
